src/main/java/com/ftw/hometerview/place/controller/dto/CompanyDto.java

```diff
@@ -0,0 +1,44 @@
+package com.ftw.hometerview.place.controller.dto;
+
+import com.ftw.hometerview.place.domain.Company;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import lombok.Builder;
+import lombok.Getter;
+
+public class CompanyDto {
+
+    @Getter
+    public static class RegisterCompany {
+
+        @NotBlank
+        private String name;
+        @NotBlank
+        private String loadName;
+        @NotNull
+        private Double lat;
+        @NotNull
+        private Double lon;
+
+        public Company toCompany() {
+            return Company.builder()
+                .name(this.name)
+                .loadName(this.loadName)     // oo시 oo구 oo동 ~~
+                .province(this.loadName.split(" ")[1])
+                .lat(this.lat)
+                .lon(this.lon)
+                .build();
+        }
+    }
+
+    @Getter
+    @Builder
+    public static class Meta {
+
+        private String companyId;
+        private String name;
+        private String loadName;
+        private String station;
```

Comment:
```suggestion
        private String nearestStationName;
```
의미하는 바가 `가까운 역 이름`이 맞다면 `nearestStationName`으로 명시하는 것은 어떨까해서 의견 제시 해봅니다. 너무 구체적인 네이밍이라고 생각되시면 해당 suggestion은 resolution 처리 해주세요